com/truckmuncher/api/trucks.proto

```diff
@@ -77,6 +86,18 @@ message ActiveTrucksResponse {
          * Value in the range [-180, 180]
          */
         required double longitude = 3;
+        /** 
+         * Indicates whether the truck's location is verified or not.
+         */
+        optional bool verified = 4 [default = false];
```

Comment:
Is this the "verified" badge we discussed the other day? Or can you also be verified with X number of upvotes, such as 100 upvotes?